library/swift/src/RetryPolicyMapper.swift

```diff
@@ -38,6 +38,7 @@ extension RetryPolicy {
       retryOn: headers.value(forName: "x-envoy-retry-on")?.compactMap(RetryRule.init) ?? [],
       retryStatusCodes: headers.value(forName: "x-envoy-retriable-status-codes")?
         .compactMap(UInt.init) ?? [],
+      // TODO: is the flatmap to apply the constructor even if there is only one value?
```

Comment:
https://www.hackingwithswift.com/articles/205/whats-the-difference-between-map-flatmap-and-compactmap